### PR TITLE
Chapter3 / HW 10

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: Src/BootCamp.Chapter/Car.cs [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
-
-namespace BootCamp.Chapter
+﻿namespace BootCamp.Chapter
 {
     // This should have the following attribute
-    // [Textable]
+    [Textable]
     public class Car
     {
+        [Textable]
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Car.cs:7]: You don't format individual properties in their own boxes, just the whole car.

File: Src/BootCamp.Chapter/TextBoxPrinter.cs [hunk 2]
```diff
@@ -1,12 +1,30 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-
-namespace BootCamp.Chapter
+﻿namespace BootCamp.Chapter
 {
     public static class TextBoxPrinter
```
Comment (Almantask) [inline Src/BootCamp.Chapter/TextBoxPrinter.cs:3]: I actually like the way you split it a lot. Having a break definitely helped you a lot 😄

**Review by Almantask — APPROVED** [review]
Looks great!

File: Src/BootCamp.Chapter/Car.cs (hunk 1 above)
Comment (ghost) [reply Src/BootCamp.Chapter/Car.cs:7]: hmm, I did but then the attribute did not work at all

File: Src/BootCamp.Chapter/TextBoxPrinter.cs (hunk 2 above)
Comment (ghost) [reply Src/BootCamp.Chapter/TextBoxPrinter.cs:3]: Thanks

File: Src/BootCamp.Chapter/Car.cs (hunk 1 above)
Comment (ghost) [reply Src/BootCamp.Chapter/Car.cs:7]: then this line gives null :  `requiredAttributeTextable = AttributesGetter.GetPropertyAttribute<Textable>(prop); `

**Pull request closed without merging**